ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,213 @@
+                            <origin>
+                                <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  This manuscript was collected by <persName
+                                    ref="PRS1071dAbbadi">A.d'Abbadie</persName> in Ethiopia.  </origin>
+                            <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
+                                <date>1902</date>.</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">119-120</citedRange>
+                                            <citedRange unit="number">87</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Chaine1912Catalogue"/>
+                                            <citedRange unit="page">56</citedRange>
+                                            <citedRange unit="number">87</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Abbadie1859Cat"/>
+                                            <citedRange unit="page">98-99</citedRange>
+                                            <citedRange unit="number">87</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Chants"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Liturgy"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="NV" when="2024-09-02">Described the manuscript based on the available descriptions.</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
```

Comment:
is it relevant for the manuscript or for the text?

Reply:
for the content of the manuscript

Reply:
then it should go to the record about the relevant work